packages/algolia-search/plugins/AlgoliaPageSearchResults.tsx

```diff
@@ -34,19 +34,19 @@ function PageHits() {
   )
 }
 
-function AlgoliaPageSearchPlugin(props: PluginProps<SearchFormProps>) {
+function AlgoliaPageSearchPlugin(props: PluginProps<CategorySearchResultsProps>) {
```

Comment:
Should we also rename the plugin to AlgoliaCategorySearchResultsPlugin?

Reply:
This plugin is still responsible for the page index so I think the current name should be more applicable